src/index.html

```diff
@@ -4,14 +4,203 @@
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset class="form-section">
+
+        <!-- Personal info -->
+
+        <legend>Personal information</legend>
+
+        <div class="form-widget">
+          <label for="surname">Surname:</label>
+          <input
+            name="surname"
+            id="surname"
+            type="text"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="name">Name:</label>
+          <input
+            name="name"
+            id="name"
+            type="text"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="age">How old are You?</label>
+          <input
+            name="age"
+            id="age"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="fullDateOfBirth">Full date of birth:</label>
+          <input
+            name="fullDateOfBirth"
+            id="fullDateOfBirth"
+            type="date"
+            min="1950-01-01"
+            max="2020-01-01"
+            required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            name="terms"
+            id="terms"
+            type="checkbox"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <!-- Registration -->
+
+      <fieldset class="form-section">
+        <legend>Registration</legend>
+
+        <div class="form-widget">
+          <label for="email">E-mail:</label>
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-widget">
+          <label for="password">Password:</label>
+          <input
+            name="password"
+            id="password"
+            type="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <!-- Facts -->
+
+      <fieldset class="form-section">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-widget">
+          <label for="catLover">Do you love cats?</label>
+
+          <input
+            name="catLover"
+            type="radio"
+            id="catLoverYes"
+            value="yes"
+            checked
+          >
+          <label for="catLoverYes">Yes</label>
+
+          <input
+            name="catLover"
```

Comment:
We will learn that a little bit later, but since the value is boolean - their names usually start with a word like `has... does...` In your case I would do it like that:
```suggestion
            name="lovesCats"
```